### DISCUSSION TITLE
Admin user was deleted. How to create a new one?

### DISCUSSION THREAD

**Discussion by liefersfl** [keycloak/keycloak#36145] [Q&A]
I'm using keycloak version `22.0.1` through bitnami helm chart `16.0.3`. 
The admin user was created by the environment varialbes `KEYCLOAK_ADMIN_PASSWORD` and `KEYCLOAK_ADMIN`. The username is not `admin` but something else.
For more than a year i could login into the `master` realm and manage keycloak using this user. All of a sudden i could not login anymore and when viewing the user_entity table in my postgres database, the admin user is not there anymore.
Regardless of why the admin user is no longer there (which is quite interesting)...how can I create a new admin user without an admin user?

**Comment by jonkoops** [comment]
See the documentation on [admin bootstrap and recovery](https://www.keycloak.org/server/bootstrap-admin-recovery).
